docs/analysis.rst

```diff
@@ -10,25 +10,143 @@ the final free energy output that you get. In theory, we could just print out
 
-**************
-Analyzing data
-**************
+* :ref:`Command line auto-execution <auto_analyze>`
+* :ref:`Cisually with Jupyter Notebooks<visual_analyze>`
+* :ref:`Full feature API <programmatic_analyze>`
 
-To perform a simple analysis of the data generated by YANK, you can use the ``yank analyze`` facility, even when YANK is running.
 
-.. code-block:: bash
+Analysis Theory
+---------------
+
+We run YANK simulations through  improve the estimate in free energy and other thermodynamic properties. We document them in
+the :doc:`algorithms` page, but summarize them here, and linked to their detailed explanation in the appropriate page.
+When running :ref:`automatically <auto_analyze>`, these steps are run in order to yield the final free energy estimate.
+The analysis procedure is carried out for each phase of the YANK simulation.
+
+
+Equilibration and Decorrelation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We first discard non-equilibrated samples and then subsample to remove correlation effects between the remaining
+samples. The :ref:`timeseries to carry out this analysis from replica exchange <repex_timeseries>` is chosen as the sum
+of potential energies from each sample, evaluated in state it was drawn from. The timeseries is run through the
+``detectEquilibration`` routine of the
+`timeseries module of PyMBAR <https://pymbar.readthedocs.io/en/master/timeseries.html>`_ to determine the correlation
+times. The :ref:`decorrelation rate is analyzed at each point in the timeseries <autocorrelate_algorithm>` by assuming
+all samples before that point are part of the equilibration time, then the number of remaining samples are computed
+by subsampling at the decorrelation rate. The point in the timeseries which preserves the maximum number of samples
+is treated as the sample at which equilibration is complete, and the remaining subsampled data are passed onto MBAR
+for free energy estimate.
+
+
+Replica Mixing Quality
+^^^^^^^^^^^^^^^^^^^^^^
+
+Quality free energy estimates require a sufficient mixing of replicas to
+`improve phase space overlap <http://www.alchemistry.org/wiki/Free_Energy_Fundamentals#Facts_from_the_Free_Energy_Difference_Definition>`_.
+Replica exchange simulations enhance sampling by allowing configurations to swap into new states, reducing energetic or steric barriers.
+However, if no or little swaps occur, then the replica exchange provides little additional benefit over single simulations.
+Furthermore, there is a good chance you have low phase space overlap between states the replicas can sample, which may
+yield not only an incorrect free energy estimate, but also underestimate the error in the free energy as well.
+
+.. note::
+
+    Terminology Note: Replica is the time-continuous trajectory of particles and box vectors. A (thermodynamic) state is
+    the collection of forces and rules which dictate how particles interact with each other and the surrounding bulk
```

Comment:
Might be useful to mention that the thermodynamic states includes temperature, pressure, Hamiltonian.